cypress/integration/pages/home.spec.js

```diff
@@ -64,4 +64,33 @@ context('Index page', () => {
     });
 
   });
+
+  describe("Topic Explorer", () => {
+    it("can show the topic explorer", () => {
+      cy.contains('How can we help?').should('be.visible')
+    })
+
+
+    it("can show food example prompts", () => {
+      cy.get('[data-search-term=food]').click();
+      cy.get('input').should('have.value', 'food')
+      cy.contains('Are you able to shop for food?')
```

Comment:
I'm worried that these rely on the hard-coded prompts we've got - when we switch to airtable they'll break for the "wrong" reason.

Reply:
(I'm not against cypress tests, but I'm interested to hear if you think this is worth the tradeoff)

Reply:
I was also thinking about this- would it be sufficient to just make an assertion on the input value because were testing the other behaviour at the unit test level?

Reply:
I have now removed the assertion around the prompts

Reply:
This feels OK to me - I like these tests as just "the page looks right, right?"-style assertions.